### ISSUE TITLE
The normalised VGG-19 model can also be found in this link.

### ISSUE THREAD

**Issue by HamidSayeed** [JunbinWang/Tensorflow-Style-Transfer-with-Adain#2]
I want to try ur implementation of arbitrary neural style transfer but cant find the link for normalized VGG-19 model .please help

**Comment by JunbinWang** [comment]
> I want to try ur implementation of arbitrary neural style transfer but cant find the link for normalized VGG-19 model .please help



Try this link：

https://github.com/JunbinWang/pretrained-models/tree/master/models/style_transfer_models_and_examples

If this project helps you, you are very welcome to leave a star : )

**Comment by HamidSayeed** [comment]
Hi Wang , thank you for ur response .
I tried the link specified , but I'm getting error , after loading the decoder npz file , it seems there are no arr_0 ,.. files inside it . 
any help please.

**Comment by JunbinWang** [comment]
> Hi Wang , thank you for ur response .
> I tried the link specified , but I'm getting error , after loading the decoder npz file , it seems there are no arr_0 ,.. files inside it .
> any help please.

Hi Hamid,

You can try this link: https://github.com/tensorlayer/adaptive-style-transfer
This is also created by me using TensorLayer. I think it works

**Comment by ChauvinisticJonatha** [comment]
> > Hi Wang , thank you for ur response .
> > I tried the link specified , but I'm getting error , after loading the decoder npz file , it seems there are no arr_0 ,.. files inside it .
> > any help please.
> 
> Hi Hamid,
> 
> You can try this link: https://github.com/tensorlayer/adaptive-style-transfer
> This is also created by me using TensorLayer. I think it works

Hi， this download page is 404， can you tell me a new page? Thank you。 I have given you a star。
